### PR TITLE
Fix typos in MainThread.cs

### PR DESCRIPTION
**Pull request by ivosnoza**
### Description of Change ###

Fix typos aysnc -> async in MainThread.cs

### Bugs Fixed ###

- Related to issue #

No ticket, just documentation fix

### API Changes ###

None

### Behavioral Changes ###

None

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Has samples (if omitted, state reason in description)
- [x] Rebased on top of `main` at time of PR
- [x] Changes adhere to coding standard
- [x] Updated documentation ([see walkthrough](https://github.com/xamarin/Essentials/wiki/Documenting-your-code-with-mdoc))

### CODE REVIEW

File: docs/en/Xamarin.Essentials/FilePicker.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
       </Parameters>
       <Docs>
         <param name="options">File picker options to use; may be null</param>
-        <summary>Starts file picker for selecting a single file.</summary>
+        <summary>Starts file picker for selecting files.</summary>
```
Comment (jfversluis) [inline docs/en/Xamarin.Essentials/FilePicker.xml:35]: I think you changed the wrong one here. The one down below seems the one that is actually wrong on line 63

**Review by jfversluis — CHANGES_REQUESTED** [review]
Just the one thing, otherwise: thank you for doing this!

**Comment by ivosnoza** [comment]
> @microsoft-github-policy-service agree

**Review by jfversluis — APPROVED** [review]

**Comment by jfversluis** [comment]
Thank you so much!

**Pull request merged**